sysobject.ids

```diff
@@ -9195,8 +9265,63 @@
 41112.1.9	Ubiquiti	NETWORKING	UniTel
 
 47196	Hewlett-Packard	NETWORKING
-47196.4.1.1.1.1	Hewlett-Packard	NETWORKING	Aruba 8400
-47196.4.1.1.1.2	Hewlett-Packard	NETWORKING	Aruba 8320-48G-SFP+
+47196.4.1.1.1.1	Hewlett-Packard	NETWORKING	Aruba  JL375A 8400 8-slot
+47196.4.1.1.1.2	Hewlett-Packard	NETWORKING	Aruba  JL479A 8320 48p 10G SFP/SFP+ and 6p 40G QSFP+ 
```

Comment:
I'm not sure this is good here.

IMHO, the purpose of the last field is just to define the product model.

First, you're adding 2 spaces after 'Aruba'.
Then you're including what I guess is better the PartNumber.
And then you're setting what I think is a long description of the device.

I can understand this can help you for your needs, but I think this is too much information for just a "device model" string.

Do you know if the PartNumber is available on a private OID ?